reframe/views/profile/rightpanel/RightPanel.js

```diff
@@ -21,24 +21,23 @@ const RightPanel = props => {
     );
   };
 
-  const compare = (a, b) => {
-    if (a.total_score < b.total_score) {
-      return 1;
-    }
-    if (a.total_score > b.total_score) {
-      return -1;
-    }
-    return 0;
-  };
-
   const repos = [];
 
   if (props.contribs) {
     const contribs = Object.values(props.contribs.repos);
-    contribs.sort(compare);
+    contribs.sort(getContribDisplayOrder);
 
     const uniqueNames = [];
-    for (const contrib of contribs) {
+    for (const i in contribs) {
+      const contrib = contribs[i];
+
+      // Don't include repos where user has made 0 commits. This happens when a user
```

Comment:
- strictly speaking one is a contributor only if the PR get's merged (I believe that's what GitHub does as well when it shows the "collaborator" badge)
 - I've some PRs that are hanging for a very long time (E.g. https://github.com/facebook/create-react-app/pull/4047).
 - How about declined PRs, are they included in `contribs` as well?

I'd vote for only including when the PR is merged. But I'm ok with resurrect and if you want I can resurrect it. (I'd need to check where 0 commits contrib lead to problems in the calculations)

Reply:
Another thing to keep in mind is that we will sometimes falsely report 0% because of [this optimization](https://github.com/ghuser-io/db/issues/10).

> I've some PRs that are hanging for a very long time

that's exactly what I meant: this PR is hanging and it's not your fault, so you shouldn't be penalized :)

Reply:
My PR will most likely never get merged. So I feel like it shouldn't be considered a contrib.

I'm not religious about it. Your shot:)

Reply:
I have the same issue with:
* ("abandonned" project) https://github.com/phillipadsmith/awesome-github/pull/36
* (declined PR) https://github.com/igrigorik/gharchive.org/pull/195

But you're right, these 2 cases are as well unimportant for me, let's give a try at your version :)